cypress/support/commands.js

```diff
@@ -296,24 +225,13 @@ Cypress.Commands.add('doYouHaveDegree', (degree) => {
 	cy.clickOnContinueButton();
 });
 
-Cypress.Commands.add('selectWhatSubjectIsYourDegree', (subject) => {
-	var subject = subject.split('-');
-	switch (subject[0]) {
-		case 'Studying':
-			cy.get(
-				'#teacher-training-adviser-steps-what-subject-degree-degree-subject-field'
-			).select(subject[1]);
-			break;
-		case 'Secondary':
-			cy.get(
-				'#teacher-training-adviser-steps-degree-stage-interested-teaching-preferred-education-phase-id-222750001-field'
-			).click();
-			break;
-		default:
-			cy.get(
-				'#teacher-training-adviser-steps-what-subject-degree-degree-subject-field'
-			).select(subject[0]);
-	}
+Cypress.Commands.add("selectWhatSubjectIsYourDegree", (subject) => {
+	cy.contains("What subject is your degree?")
+		.invoke("attr", "for")
+		.then(function (val) {
+			let id = "#" + val;
+			cy.get(id).select(subject);
```

Comment:
I'd be inclined to just interpolate where you have these for brevity:

```
cy.get(`#${id}`).select(subject);
```

Reply:
Resolved